api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -110,6 +127,14 @@ public static Tracer getTracer(String instrumentationName, String instrumentatio
     return get().getTracer(instrumentationName, instrumentationVersion);
   }
 
+  /**
+   * Unsets the global {@link OpenTelemetry}. This is only meant to be used from tests which need to
+   * reconfigure {@link OpenTelemetry}.
+   */
+  public static void resetForTest() {
+    GlobalOpenTelemetry.globalOpenTelemetry = null;
```

Comment:
why prefix `globalOpenTelemetry` with the class name here, but not other places? Is this an artifact of it being moved in & out of this class?

Reply:
Yes :)